src/containers/Header/PreviewFlag.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+export default function PreviewFlag({ name }) {
+
+    if(!name) {
+        return null
+    }
+
+  return (
```

Comment:
@queuing4oranges Please update indentation. We use `tabs` which have a size of 4 spaces

Reply:
and apply it to the whole file ,)